lib/reader.ts

```diff
@@ -258,18 +268,18 @@ class ParquetReader {
    * Returns the user (key/value) metadata for this file
    */
   getMetadata() {
-    let md = {};
-    for (let kv of this.metadata.key_value_metadata) {
+    let md: Record<string, unknown> = {};
+    for (let kv of this.metadata!.key_value_metadata!) {
       md[kv.key] = kv.value;
     }
 
     return md;
   }
 
-  exportMetadata(indent) {
-    function replacer(key, value) {
+  exportMetadata(indent: string | number | undefined) {
+    function replacer(key: any, value: parquet_thrift.PageLocation | bigint | {[string:string]: any}) {
```

Comment:
Key here is not used at all so I left it as any

Reply:
I think this might work then as well:
```suggestion
    function replacer(_key: unknown, value: parquet_thrift.PageLocation | bigint | {[string:string]: any}) {
```